gcp/api/server.py

```diff
@@ -1064,7 +1066,10 @@ def query_by_version(context: QueryContext,
       for bug in new_bugs:
         if bug not in bugs:
           bugs.append(bug)
-
+    elif ecosystem_info.supports_comparing:
```

Comment:
This needs to be a variable like is_semver above where we check if ecosystem_info is None or not.